### PR TITLE
[KEYCLOAK-8652] [KEYCLOAK-8480] [KEYCLOAK-8606] RH-SSO 7.3 OpenShift image updates for rebase on top of EAP CD 14

### PR DESCRIPTION
**Pull request by iankko**
* **[KEYCLOAK-8652] Synchronize [standalone-openshift.xml](https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso-cd-dev/modules/sso/config/launch/setup/73/added/standalone-openshift.xml) with standalone-ha.xml**
  from current Keycloak master (4.6.0.Final-SNAPSHOT) upstream version
    
  Port definition of replicated caches into the form expected by Wildfly 14
  Modify OPENSHIFT log formatter into the form expected by EAP CD 14 image

* **[KEYCLOAK-8480] Replace jolokia, logging, and openshift EAP-7.1 cct_modules**
     with their EAP CD 14 counterparts

* **[KEYCLOAK-8606] Use jgroups and elytron config modules from EAP CD 14**
    
     Also add:
     * ```EJB_APPLICATION_SECURITY_DOMAINS```,
     * ```ELYTRON_INTEGRATION```, and
     * ```HTTP_APPLICATION_SECURITY_DOMAINS```
    
     <br/>
     XML anchors, which are configurable via these two modules

Signed-off-by: Jan Lieskovsky <jlieskov@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by iankko** [comment]
@luck3y @rcernich @Crumby @pdrozd PTAL (once got a chance)

Thank you, Jan

File: image.yaml [hunk 1]
```diff
@@ -68,14 +72,14 @@ modules:
           - name: os-eap7-openshift
           - name: os-eap-s2i
           - name: os-java-jolokia
-          - name: jolokia
+          - name: jboss.eap.cd.jolokia
           - name: os-eap7-openshift
-          - name: os-eap7-modules
+          - name: jboss.eap.cd.openshift.modules
```
Comment (luck3y) [inline image.yaml:77]: You may want to include the jboss-eap-config-jgroups and elytron modules as well, like here:

https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/eap-cd/image.yaml#L56

These update the elytron and jgroups configs to use a non-deprecated configuration.

**Review by luck3y — APPROVED** [review]
LGTM Jan, with just the note about JGroups & Elytron above.

File: image.yaml (hunk 1 above)
Comment (iankko) [reply image.yaml:77]: @luck3y Thanks, Ken! Included those (together with the metrics & tracing ones), see last commit.

**Review by luck3y — APPROVED** [review]

**Comment by iankko** [comment]
@luck3y @rcernich Updated the (previous form of the) PR to add / re-use just the following EAP CD modules:

- ```jboss.eap.cd.jolokia```,
- ```jboss.eap.cd.openshift.modules```,
- ```jboss.eap.cd.logging```,
- ```jboss.eap.config.jgroups```,
- ```jboss.eap.config.elytron```

Other ones (previously present, but not applicable to RH-SSO image) were removed. Also updated the ```standalone-openshift.xml`` file to list only new XML / OpenShift anchors, that are actually reachable / configurable via some of these modules.

PTAL (once got a chance)

Thanks, Jan

**Comment by iankko** [comment]
@ASzc This one should be ready to go (once the corresponding [standalone one](https://github.com/jboss-container-images/redhat-sso-7-image/pull/59) got merged).

PTAL (once got a chance)

Thanks, Jan

**Comment by iankko** [comment]
Merging, so we can rebuild the image.

**Pull request merged**